### PR TITLE
Dockerfile.alpine: add Git & Mercurial

### PR DESCRIPTION
**Pull request by Dominik-K**
Fixes partly #897

### CODE REVIEW

File: build/Dockerfile.alpine [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM golang:1.13-alpine
 
-# required to support cgo
-RUN apk --no-cache add gcc musl-dev
+# needed most times for `go get`, etc. See https://github.com/docker-library/golang/issues/80
+RUN apk add --no-cache git mercurial
```
Comment (jirfag) [inline build/Dockerfile.alpine]: why did you remove `gcc` and `musl-dev`?

Comment (Dominik-K) [reply build/Dockerfile.alpine]: @jirfag The `golang:alpine` images already install them. See e.g. https://github.com/docker-library/golang/blob/7b69aab8c4a9117743168ca8b19b314c0307e415/1.13/alpine3.11/Dockerfile#L16-L17
(noted it also in the commit message, though)

Comment (shogo82148) [reply build/Dockerfile.alpine]: `gcc` and `musl-dev` are only used for building Go, and they are removed from the `golang:alpine` image at https://github.com/docker-library/golang/blob/7b69aab8c4a9117743168ca8b19b314c0307e415/1.13/alpine3.11/Dockerfile#L55

Actually, running `gcc` fails with the `golang:alpine` image.

```
$ docker run golang:1.13-alpine gcc
docker: Error response from daemon: OCI runtime create failed: container_linux.go:346: starting container process caused "exec: \"gcc\": executable file not found in $PATH": unknown.
ERRO[0000] error waiting for container: context canceled
$ docker run golang:1.13-alpine bash
docker: Error response from daemon: OCI runtime create failed: container_linux.go:346: starting container process caused "exec: \"bash\": executable file not found in $PATH": unknown.
ERRO[0000] error waiting for container: context canceled
```

Comment (Dominik-K) [reply build/Dockerfile.alpine]: @shogo82148 Thanks for the clarification. 
@jirfag I've undone this commit. Sorry about my mistake.
